lib/chewy/rspec/update_index.rb

```diff
@@ -250,4 +244,17 @@ def array_difference(first, second)
     end
     difference.none?
   end
+
+  class Collector
+    attr_reader :updates
+
+    def initialize
+      @updates = []
+    end
+
+    def perform(body)
```

Comment:
It is used to collect data only. It mimics under the `Chewy::Type::Import::Bulk` class, repeats its API and returns empty hash to show that everything is fine. It is some kind of a mock.